src/Config/ObjectModel/EntityRestOptions.cs

```diff
@@ -16,7 +16,7 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 /// <param name="Methods">The HTTP verbs that are supported for this entity.</param>
```

Comment:
As an addition to the param summary, it would help to add that Methods only hold relevance for stored procedures. Won't cause confusion to someone new looking at the code about the purpose of this property.